mix.exs

```diff
@@ -46,7 +46,9 @@ defmodule Beeline.MixProject do
     [
       {:gen_stage, "~> 1.0"},
       {:nimble_options, "~> 0.1"},
-      {:kelvin, "~> 0.3", optional: true},
+      {:jason, "~> 1.0"},
+      {:health_checker, "~> 2.2", organization: "cuatro"},
```

Comment:
we can't publish this package publicly until we get rid of this dep

I'm thinking about rolling health_checker into this library and making new appsignal+honeycomb exporter libraries. It'd be a good bit of work but probably worth it to be able to keep everything open-sourceable

I could also be interested in re-naming health_checker. The main reason I see not to just go ahead and publish that publicly now is that it doesn't really deserve to squat on the name health_checker (IMHO) in the public hex.

Reply:
indeed. it is not as general-purpose as the name suggests. time to tony-fy that name or encode it with some cleverness algorithm.